utils/drl_algorithm_selector.py

```diff
@@ -0,0 +1,25 @@
+from stable_baselines3 import PPO, DQN, A2C
+
+class ModelSelector:
+    def __init__(self, model_type, env, config):
+        self.model_type = model_type
```

Comment:
Consider refactoring this method to be more modular by using helper methods or a dictionary to map 'model_type' to configurations and constructors.

Reply:
For example:
class ModelType(Enum):
    PPO = 'PPO'
    DQN = 'DQN'
    A2C = 'A2C'